### PR TITLE
BUG: Fix ExtractContourWithSnakes example errors.

### PR DESCRIPTION
**Pull request by jhlegarreta**
* Exit with failure if the number of arguments is not enough.
* Remove the unused exception argument `n`.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
@hjmjohnson @thewtex @blowekamp @fbudin69500 @dzenanz please review and merge if appropriate.

File: ImageSegmentation/ExtractContourWithSnakes.cxx [hunk 1]
```diff
@@ -47,7 +47,7 @@ int main( int argc, char* argv[] )
     std::cout << "Usage " << argv[0]
               << " points alpha beta gamma sigma iterations [image]"
               << std::endl;
-    return EXIT_SUCCESS;;
+    return EXIT_FAILURE;;
```
Comment (thewtex) [inline ImageSegmentation/ExtractContourWithSnakes.cxx]: Side-note: there is an extra semi-colon here

File: ImageSegmentation/ExtractContourWithSnakes.cxx [hunk 2]
```diff
@@ -103,7 +103,7 @@ int main( int argc, char* argv[] )
     {
     P = computeP(alpha, beta, gamma, N);
     }
-  catch (int n)
+  catch (...)
     {
     return EXIT_FAILURE;;
```
Comment (thewtex) [inline ImageSegmentation/ExtractContourWithSnakes.cxx]: Side-note: there is an extra semi-colon here

**Comment by thewtex** [comment]
Please rebase on master to address merge conflicts.

File: ImageSegmentation/ExtractContourWithSnakes.cxx (hunk 1 above)
Comment (jhlegarreta) [reply ImageSegmentation/ExtractContourWithSnakes.cxx]: Done in 3764fad.

File: ImageSegmentation/ExtractContourWithSnakes.cxx (hunk 2 above)
Comment (jhlegarreta) [reply ImageSegmentation/ExtractContourWithSnakes.cxx]: Fixed once PR #14 was merged and having rebased on top of master: 3764fad.

**Comment by jhlegarreta** [comment]
3764fad rebased on master.

**Comment by jhlegarreta** [comment]
@thewtex This can also be merged.

**Pull request merged**